xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/displayers/displayerMixin.js

```diff
@@ -73,27 +74,11 @@ export default {
   },
 
   methods: {
-    // This method should be used to apply edit and go back to view mode
-    // It validate the entered value, ensuring that is is valid for the server
-    applyEdit (newValue) {
-      this.logic.setValue({
-        entry: this.entry,
-        propertyId: this.propertyId,
-        value: newValue
-      });
-      // Go back to view mode
-      // (there might be a cleaner way to do this)
-      this.$el.__vue__.view();
-    },
-
-    // This method should be used to cancel edit and go back to view mode
-    // This is like applyEdit but it does not save the entered value
-    cancelEdit () {
-      // Go back to view mode
-      // (there might be a cleaner way to do this)
-      this.$el.__vue__.view();
-    },
-
-  },
-
+    /**
+     * Generic save operation where the saved value is the 
+     */
+    genericSave() {
+      this.editBus.save(this.entry, this.propertyId, [{[this.propertyId]: this.editedValue}])
```

Comment:
Why are you sending an array of objects? I noticed this in the ``xobjectProperty`` displayer also. As I said there it would have made sense if the objects had all the same structure. But it's not the case. I find this overly complicated.